R/pkg/R/serialize.R

```diff
@@ -83,6 +83,7 @@ writeObject <- function(con, object, writeType = TRUE) {
          Date = writeDate(con, object),
          POSIXlt = writeTime(con, object),
          POSIXct = writeTime(con, object),
+         bigint = writeDouble(con, object),
```

Comment:
and another thing, there is no `bigint`  in R
so I'm not sure how we would hit this path